python/ray/_private/ray_constants.py

```diff
@@ -89,6 +89,7 @@ def env_bool(key, default):
 DEFAULT_DASHBOARD_PORT = 8265
 DASHBOARD_ADDRESS = "dashboard"
 PROMETHEUS_SERVICE_DISCOVERY_FILE = "prom_metrics_service_discovery.json"
+DEFAULT_DASHBOARD_AGENT_LISTEN_PORT = 52365
```

Comment:
Can we add a regression test to make sure the `DEFAULT_DASHBOARD_AGENT_LISTEN_PORT` is `52365`? There's a lot of defaults in the code and the documentation that make this assumption, so it would be nice to add a short test that looks something like:

```python
def test_default_dashboard_agent_listen_port():
    """Defaults in the code and the documentation assume the dashboard agent listens to HTTP on port 52365."""
    assert DEFAULT_DASHBOARD_AGENT_LISTEN_PORT == 52365
```